messaging/src/main/java/org/axonframework/messaging/commandhandling/annotation/Command.java

```diff
@@ -55,9 +55,10 @@
     /**
      * The business or domain name of the command.
      * <p>
-     * Will typically be mapped to a {@link QualifiedName#QualifiedName(String)} and inserted into a
-     * {@link MessageType}. By using the String-based constructor of the {@link QualifiedName}, this field  will
-     * represent the combination of the {@link QualifiedName#localName()} and {@link QualifiedName#namespace()},
+     * Is used to define the {@link QualifiedName#localName()} of a fully qualified name.
+     * <p>
+     * Defaults to the simple name of the annotated class. Note that when an inner class is annotated, the simple name
+     * does not include the names of the parent classes.
```

Comment:
Nit: nested class is the term for both inner and static nested classes; enclosing class is the name of the container class (parent class = super class).
```suggestion
     * Defaults to the simple name of the annotated class. Note that when a nested class is annotated, the simple name
     * does not include the names of the enclosing classes.
```